### PR TITLE
add takephotos coords

### PR DESCRIPTION
**Pull request by evanillaa**

### CODE REVIEW

File: client/main.lua [hunk 1]
```diff
@@ -157,12 +174,12 @@ AddEventHandler('prison:client:Leave', function()
 	end
 end)
 
-RegisterNetEvent('prison:client:UnjailPerson')
+RegisterNetEvent('prison:client:UnjailPerson') ---- We need someone to help write the code to return the clothes to the player after getting out of prison ---
```
Comment (FreakOutModz) [inline client/main.lua:177]: Do not comment in PR's, do it in the github. You should remove it and request it in the comments of the PR.

File: client/main.lua [hunk 2]
```diff
@@ -246,3 +263,83 @@ function DrawText3D(x, y, z, text)
...
+    TriggerEvent('InteractSound_CL:PlayOnOne', 'jaildoor', 1.0)
+end
+
+function JailIclothes()
+	local ped = ped
+	local model = GetEntityModel(ped)
+	local clothesrand = math.random(1,2)
+    if model == `mp_f_freemode_01` then
+		if clothesrand == 1 then
+			ClearPedProp(ped, 0)
```
Comment (FreakOutModz) [inline client/main.lua:306]: Instead of `ClearPedProp(ped, 0)` everywhere, you should add it after the model check.

File: client/main.lua [hunk 3]
```diff
@@ -246,3 +263,83 @@ function DrawText3D(x, y, z, text)
...
+		else
+			ClearPedProp(ped, 0)
+			SetPedComponentVariation(ped, 3, 5, 0, 2) -- Arms
+			SetPedComponentVariation(ped, 4, 38, 0, 2) -- Leg
+			SetPedComponentVariation(ped, 5, -1, 0, 2) -- Parachute / bag
+			SetPedComponentVariation(ped, 6, 6, 10, 2) -- Shoes
+			SetPedComponentVariation(ped, 8, 15, 0, 2) -- Undershirt
+			SetPedComponentVariation(ped, 9, -1, 0, 2) -- Kevlar
+			SetPedComponentVariation(ped, 11, 5, 0, 2) -- Torso 2
+		end
```
Comment (FreakOutModz) [inline client/main.lua:343]: For a cleaner code this should be:


```lua
if model == `mp_f_freemode_01` then
    ClearPedProp(ped, 0)
    SetPedComponentVariation(ped, 6, 27, 11, 2) -- Shoes
    SetPedComponentVariation(ped, 8, 14, 0, 2) -- Undershirt
    SetPedComponentVariation(ped, 5, -1, 0, 2) -- Parachute / bag
    SetPedComponentVariation(ped, 9, -1, 0, 2) -- Kevlar
    if clothesrand == 1 then
        SetPedComponentVariation(ped, 3, 3, 0, 2) -- Arms
        SetPedComponentVariation(ped, 4, 38, 0, 2) -- Leg
        SetPedComponentVariation(ped, 11, 67, 0, 2) -- Torso 2
    else
        SetPedComponentVariation(ped, 3, 14, 0, 2) -- Arms
        SetPedComponentVariation(ped, 4, 38, 1, 2) -- Leg
        SetPedComponentVariation(ped, 11, 84, 2, 2) -- Torso 2
    end
elseif model == `mp_m_freemode_01` then
    ClearPedProp(ped, 0)
    SetPedComponentVariation(ped, 3, 5, 0, 2) -- Arms
    SetPedComponentVariation(ped, 6, 6, 10, 2) -- Shoes
    SetPedComponentVariation(ped, 8, 15, 0, 2) -- Undershirt
    SetPedComponentVariation(ped, 5, -1, 0, 2) -- Parachute / bag
    SetPedComponentVariation(ped, 9, -1, 0, 2) -- Kevlar
    if clothesrand == 1 then
        SetPedComponentVariation(ped, 4, 38, 1, 2) -- Leg
        SetPedComponentVariation(ped, 11, 68, 0, 2) -- Torso 2
    else
        SetPedComponentVariation(ped, 4, 38, 0, 2) -- Leg
        SetPedComponentVariation(ped, 11, 5, 0, 2) -- Torso 2
    end
    
   ```

File: client/main.lua [hunk 4]
```diff
@@ -246,3 +263,83 @@ function DrawText3D(x, y, z, text)
...
+    SetEntityHeading(Ped,170.74) 
+    TriggerEvent('InteractSound_CL:PlayOnOne', 'photo', 0.4)
+    Citizen.Wait(3000) 
+     TriggerEvent('InteractSound_CL:PlayOnOne', 'photo', 0.4)
+    Citizen.Wait(3000)       
+    SetEntityHeading(Ped,270.0)
+    Citizen.Wait(2000)
+    DoScreenFadeOut(1100)   
+    Citizen.Wait(2000)
+    TriggerEvent('InteractSound_CL:PlayOnOne', 'jaildoor', 1.0)
```
Comment (FreakOutModz) [inline client/main.lua:297]: Set sound to `0.7` since `1.0` is really loud for some players.

File: client/main.lua [hunk 5]
```diff
@@ -246,3 +263,83 @@ function DrawText3D(x, y, z, text)
     DrawRect(0.0, 0.0+0.0125, 0.017+ factor, 0.03, 0, 0, 0, 75)
     ClearDrawOrigin()
 end
+
+function JailIntro()
+    local Ped = PlayerPedId()
```
Comment (FreakOutModz) [inline client/main.lua:268]: **NOT NECESSARY** but for a better code look this should all have the same name in script so `ped` and not `Ped`

**Review by FreakOutModz — CHANGES_REQUESTED** [review]

File: client/main.lua (hunk 2 above)
Comment (FreakOutModz) [reply client/main.lua:306]: I did all the code [here](https://github.com/qbcore-framework/qb-prison/pull/7#discussion_r708647219)

**Review by FreakOutModz — APPROVED** [review]
Nice work

**Comment by evanillaa** [comment]
Please do not close the proposal
I hope you can fix the errors, if any
And come out with a good text
I'm not a good expert at finishing code well
But I try to help a lot
I hope everyone will participate in solving the problems
I thank you @GhzGarage @FreakOutModz thank for all support this project

**Review by FreakOutModz — APPROVED** [review]
Changes looking good.

**Comment by evanillaa** [comment]
Hey, FreakOutModz
Long time no see your response

**Pull request closed without merging**